code/controllers/subsystem/adjacent_air.dm

```diff
@@ -0,0 +1,48 @@
+SUBSYSTEM_DEF(adjacent_air)
+	name = "Atmos Adjacency"
+	flags = SS_BACKGROUND
+	runlevels = RUNLEVEL_GAME | RUNLEVEL_POSTGAME
+	wait = 1
```

Comment:
Gonna need to test this a bit more and find a good number here, 5-10 is probably appropriate. The higher it is the more of the potential gains during explosions and such can be realized, but making it too high turns rooms into ZAS zones

Reply:
Some ZAS performance is okay, after all high pressure gas propagates extremely quickly into rooms. aka 4.500 kPa etc.

Reply:
Experiment with as high as 10-15 imo I'm interested in some very minor zone behavior